src/AI/JakeMan.java

```diff
@@ -595,13 +595,19 @@ private GameAction findTravelAction(
         break;
       }
     }
+    // If we have to move and have no destinations, make the start tile the goal
+    if( mustMove && null == goal )
+    {
+      goal = new XYCoord(unit);
+      path = GamePath.stayPut(unit);
```

Comment:
How does this let JakeMan move landers out of the way?

Reply:
Line 605 right below just returns `null` if we don't have any goals.
So if we don't have a goal, we set a goal of staying put (and fail because there's more important stuff going on).
Intuitive algorithm design, I know.

Reply:
Oh, but of course. How silly of me not to see that before.